export function notFoundError(entity = "Recurso") {
    return {
        type: "notFound",
        message: `${entity} n√£o encontrado(a)`
    }
}